airgun/views/task.py

```diff
@@ -22,6 +22,10 @@ class TaskReadOnlyEntry(ReadOnlyEntry):
     )
 
 
+class TaskPagination(Pagination):
+    PER_PAGE_BUTTON_DROPDOWN = ".//div[button[@id='tasks-table-dropdown']]"
```

Comment:
Note to self: once we start seeing more of changes like that, we should see if we can't just change Pagination to:
```python
    PER_PAGE_BUTTON_DROPDOWN = ".//div[button[contains(@id, '-dropdown')]]"
```